### DISCUSSION TITLE
Additional Frontmatter Data

### DISCUSSION THREAD

**Discussion by genepan2** [Vagr9K/gatsby-advanced-starter#153] [Q&A]
Hi,

I would like to display additional data from the frontmatter section on
a) the single post view
b) the feed view (homepage)

The a) I successfuly accomplished. But I don't understand how to do b). 

Can anyone help?

**Comment by Vagr9K** [comment]
Hello,

Depending on the starter theme you're using, you can either edit the theme code directly in `themes/THEME_NAME` or rely on [shadowing](https://www.gatsbyjs.com/docs/how-to/plugins-and-themes/shadowing/).

Your main files of interest are:
- https://github.com/Vagr9K/gatsby-advanced-starter/blob/master/themes/amaranth/src/gatsby-theme-advanced/templates/feed/index.tsx (for styling the Amaranth feed page)
- https://github.com/Vagr9K/gatsby-advanced-starter/blob/master/themes/advanced/src/templates/feed/index.tsx (for styling the base "Advanced" theme)
- https://github.com/Vagr9K/gatsby-advanced-starter/blob/master/themes/advanced/src/templates/feed/queries.ts (for querying extra data which is done at the level of the "Advanced" theme)

Also, take a look at [Using Multiple Gatsby Themes](https://www.gatsbyjs.com/docs/themes/using-multiple-gatsby-themes/) to see how  the themes are combined and used in the starter.

**Reply by genepan2** [reply]
Thank you for the reply!

In fact I was shadowing and it is strange that the post query works and the listing query doesn't.

My solution for now is shadowing the post and modifying the files in `themes/advancedsrc/src/templates/feed`. It works, but this is not an elegant solution. 

Could the problem andsolution lie in the `themes/advanced/gatsby/utils/queries.ts`?

Thanks again!

**Reply by Vagr9K** [reply]
Is it possible that Gatsby is warning you about your queries not executing?

There was a similar case here: https://github.com/Vagr9K/gatsby-advanced-starter/discussions/127

**Reply by genepan2** [reply]
Thanks, I will check!
